src/patternfly/components/Nav/nav.scss

```diff
@@ -1,276 +1,113 @@
-  --#{$nav}__subnav__link--hover--after--BorderWidth: var(--#{$pf-global}--BorderWidth--sm);
-  --#{$nav}__subnav__link--focus--after--BorderWidth: var(--#{$pf-global}--BorderWidth--sm);
-  --#{$nav}__subnav__link--active--after--BorderWidth: var(--#{$pf-global}--BorderWidth--sm);
-  --#{$nav}__subnav__link--m-current--after--BorderWidth: var(--#{$pf-global}--BorderWidth--xl);
-  --#{$nav}__subnav--MaxHeight: 0;
-  --#{$nav}__subnav__subnav--PaddingLeft: var(--#{$pf-global}--spacer--lg);
-  --#{$nav}__subnav__subnav__link--PaddingLeft: var(--#{$pf-global}--spacer--md);
-  --#{$nav}__subnav__subnav__link--FontSize: var(--#{$pf-global}--FontSize--xs);
-  --#{$nav}__item--m-expanded__subnav--MaxHeight: 100%;
-  --#{$nav}__subnav--c-divider--PaddingRight: var(--#{$pf-global}--spacer--lg);
-  --#{$nav}__subnav--c-divider--PaddingLeft: var(--#{$pf-global}--spacer--lg);
+  --#{$nav}__link--hover--BackgroundColor: var(--pf-t--global--background--color--action--plain--alt--hover);
+  --#{$nav}__link--focus--BackgroundColor: var(--pf-t--global--background--color--action--plain--alt--focus);
+  --#{$nav}__link--active--BackgroundColor: var(--pf-t--global--background--color--action--plain--alt--selected);
+  --#{$nav}__link--m-current--BackgroundColor: var(--pf-t--global--background--color--action--plain--alt--selected);
+  --#{$nav}__link--Transition: var(--#{$pf-global}--Transition);
+  --#{$nav}__link--BorderRadius: var(--pf-t--global--border--radius--small);
+  --#{$nav}__link--row-gap--offset: calc(var(--#{$nav}__list--RowGap) / 2 * -1); // block start/end offset by half row gap to consume space between items to prevent flicker
+  --#{$nav}__link--column-gap--offset: 0;
+
+  // Use these special variables to define the clickable area offset for nav links
+  --#{$nav}__link--clickable-area--inset-block-start: calc(var(--pf-v5-c-nav__item--RowGap) / 2 * -1);
+  --#{$nav}__link--clickable-area--inset-block-end: calc(var(--pf-v5-c-nav__item--RowGap) / 2 * -1);
+  --#{$nav}__link--clickable-area--inset-inline-start: 0;
+  --#{$nav}__link--clickable-area--inset-inline-end: 0;
+
+  // Horizontal
+  --#{$nav}--m-horizontal--FlexDirection: row;
+  --#{$nav}--m-horizontal--PaddingBlockStart: var(--pf-t--global--spacer--sm);
+  --#{$nav}--m-horizontal--PaddingBlockEnd: var(--pf-t--global--spacer--sm);;
+  --#{$nav}--m-horizontal__list--RowGap: 0;
+  --#{$nav}--m-horizontal__list--ColumnGap: var(--pf-t--global--spacer--xs);
+  --#{$nav}--m-horizontal__link--AlignItems: center;
+  --#{$nav}--m-horizontal__link--WhiteSpace: nowrap;
+  --#{$nav}--m-horizontal__link--row-gap--offset: 0;
+  --#{$nav}--m-horizontal__link--column-gap--offset: calc(var(--#{$nav}__list--ColumnGap) / 2 * -1); // inline start/end offset by half row gap to consume space between items to prevent flicker
+
+  // Horinzontal subnav
+  --#{$nav}--m-horizontal--m-subnav--BorderBlockStart--width: var(--pf-t--global--border--color--200);
```

Comment:
Here you have the width and color mixed up. Also, I think the color should be `--pf-t--global--border--color--default`.